docs/topics/mta-web-proxy-config.adoc

```diff
@@ -0,0 +1,20 @@
+// Module included in the following assemblies:
+//
+// * docs/cli-guide/master.adoc
+
+:_content-type: PROCEDURE
+[id="mta-web-proxy-config_{context}"]
+= Configuring HTTP and HTTPS proxy settings
+
+This management module configures HTTP & HTTPS proxy settings.
+
+.Procedure
+
+. Click the radio button to enter the following information.
+. In the left menu of the *Administrator* view, click *Proxy*.
```

Comment:
Good fromatting. It's the navigation bar, but it is so prominent, you can write . In *Administrator* view, click *Proxy*.

Reply:
. In the *Administrator* view, click *Proxy*.

Reply:
No need for "the".